README.md

```diff
@@ -188,22 +188,67 @@ realmlist.wtf
 Similar to the `ls` command with the `-l` and `-a` options, additional detailed information can be included with the `list` subcommand. The `-a` option includes printing "special" files used in MPQ archives including: `(listfile)`, `(attributes)` and `(signature)`.
 
 ```
-mpqcli list -d -a wow-patch.mpq 
-      88604  enUS  Mar 29 2006 12:02 BM_COKETENT01.BLP
-        243  enUS  Apr  5 2006 07:28 Blizzard_CraftUI.xml
-        388  enUS  Mar 30 2006 05:32 CreatureSoundData.dbc
-        ...
-        184  enUS  Apr  5 2006 07:28 Blizzard_CraftUI.lua
-      44900  enUS  Mar 29 2006 12:01 30ee7bd3959906e358eff01332cf045e.blp
-         68  enUS  Apr  7 2006 10:58 realmlist.wtf
+$ mpqcli list -d -a wow-patch.mpq 
+   88604 enUS Mar 29 2006 12:02  BM_COKETENT01.BLP
+     243 enUS Apr  5 2006 07:28  Blizzard_CraftUI.xml
+     388 enUS Mar 30 2006 05:32  CreatureSoundData.dbc
+     ...
+     184 enUS Apr  5 2006 07:28  Blizzard_CraftUI.lua
+   44900 enUS Mar 29 2006 12:01  30ee7bd3959906e358eff01332cf045e.blp
+      68 enUS Apr  7 2006 10:58  realmlist.wtf
+```
+
+### List specific properties
+
+The `list` subcommand supports listing the following properties:
+
+- `hash-index` - Index in the hash table where the file entry is.
+- `name-hash1` - The first hash of the file name.
+- `name-hash2` - The second hash of the file name.
+- `name-hash3` - 64-bit Jenkins hash of the file name, used for searching in the HET table.
+- `locale` - Locale info of the file.
+- `file-index` - Index in the file table of the file.
+- `byte-offset` - Offset of the file in the MPQ, relative to the MPQ header.
+- `file-time` - Timestamp of the file.
+- `file-size` - Uncompressed file size of the file, in bytes.
+- `compressed-size` - Compressed file size of the file, in bytes.
+- `encryption-key` - Encryption key for the file.
+- `encryption-key-raw` - Encryption key for the file.
+- `flags` - File flags for the file within MPQ:
+  * `i`: File is Imploded (By PKWARE Data Compression Library).
+  * `c`: File is Compressed (By any of multiple methods).
+  * `e`: File is Encrypted.
+  * `2`: File is Encrypted with key v2.
+  * `p`: File is a Patch file.
+  * `u`: File is stored as a single Unit, rather than split into sectors.
+  * `d`: File is a Deletion marker. Used in MPQ patches, indicating that the file no longer exists.
+  * `r`: File has Sector CRC checksums for each sector. This is ignored if the file is not compressed or imploded.
+  * `s`: Present on STANDARD.SNP\(signature).
+  * `x`: File exists; this is reset if the file is deleted.
+  * `m`: Mask for a file being compressed.
+  * `n`: Use default flags for internal files.
+  * `f`: Fix key; This is obsolete.
```

Comment:
The flags and descriptions are from here: https://github.com/ladislav-zezula/StormLib/blob/b62de3c83fc146c4e8a05bde15d39e19588c28a4/src/StormLib.h#L220-L238